db/subplugins.php

```diff
@@ -13,18 +13,22 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
-
 /**
  * Surveypro subplugin types declaration
  *
  * @package   mod_surveypro
  * @copyright 2013 onwards kordan <kordan@mclink.it>
  * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
-
 defined('MOODLE_INTERNAL') || die();
-
 $subplugins = array('surveyprofield' => 'mod/surveypro/field',
                     'surveyproformat' => 'mod/surveypro/format',
                     'surveyprotemplate' => 'mod/surveypro/template',
                     'surveyproreport' => 'mod/surveypro/report');
+
+// TODO: this file is here because of compatibility with versions earlier than 3.7.
+// Remember to drop it once $plugin->requires will be set to require Moodle 3.8 in versione.php.
+// As debugging tool, instead of dropping it, you should consider the replacement of the code with the following one:
```

Comment:
And, once mod_surveypro require Moodle 3.8... the best thing we can do is to remove the ````subplugins.php```` file completely. No need for it anymore (in fact we have decided to remove all the core ones in master this week - instead of keeping any debugging)